frameworks/kafka/cli/dcos-kafka/main.go

```diff
@@ -5,11 +5,12 @@ import (
 	"github.com/mesosphere/dcos-commons/cli"
 	"gopkg.in/alecthomas/kingpin.v2"
 	"log"
-	"strings"
+	"net/url"
+	"strconv"
 )
 
 func main() {
-	app, err := cli.NewApp("0.1.0", "Mesosphere", "Provides an example for DC/OS service developers")
+	app, err := cli.NewApp("0.1.0", "Mesosphere", "Deploy and Manage Apache Kafka framework")
```

Comment:
The Kafka cli doesn't deploy Kafka.  `Manage the Apache Kafka DC/OS service` maybe.